dockers/docker-database/Dockerfile.j2

```diff
@@ -35,6 +35,15 @@ RUN sed -ri 's/^(save .*$)/# \1/g;                       \
              s/^client-output-buffer-limit pubsub [0-9]+mb [0-9]+mb [0-9]+/client-output-buffer-limit pubsub 0 0 0/ \
             ' /etc/redis/redis.conf
 
-COPY ["supervisord.conf", "/etc/supervisor/conf.d/"]
-
-ENTRYPOINT ["/usr/bin/supervisord"]
```

Comment:
>supervisord [](start = 22, length = 11)

We prefer keep supervisord as entrypoint. You may refer other docker containers for example with a start.sh.

Reply:
if the database_config file is static as you said, then we can use supervisord as Entrypoint, and we can generate supervisord.conf during compiling/building

Reply:
Keeping supervisord as entrypoint is a good practice.  Even with database_config.json, it shall be possilble to generate  redis-server.sh from redis-server.sh.j2 within db-docker-init.sh(dockers/docker-database/docker_init.sh), and let redis-server.sh handle the dynamic db instances case?

```
[program:db-docker-init.sh]
command=/usr/bin/db-docker-init.sh
priority=2
autostart=true
autorestart=false
stdout_logfile=syslog
stderr_logfile=syslog

[program:redis-server.sh]
command=/usr/bin/redis-server.sh
priority=3
autostart=false
autorestart=false
stdout_logfile=syslog
stderr_logfile=syslog
```